opentracing-spring-web/src/test/java/io/opentracing/contrib/spring/web/client/TracingAsyncRestTemplateTest.java

```diff
@@ -40,4 +50,42 @@ public AsyncRestTemplate template() {
 
         mockServer = MockRestServiceServer.bindTo(client.template()).build();
     }
+
+    @Test
+    public void testMultipleRequests() throws InterruptedException {
```

Comment:
I'm not sure what exactly you are requesting.

Test executes multiple async requests and tests if each span representing a given request is linked with a parent span. 


Edit: Now test creates a parent span for each async reques.